Makefile

```
@@ -21,14 +26,27 @@ clean: ## Remove build artifacts
-	RUSTFLAGS="$(RUST_FLAGS)" $(CARGO) test $(TEST_FLAGS) $(BUILD_JOBS)
+	RUSTFLAGS="$(RUST_FLAGS)" $(CARGO) $(TEST_CMD) $(TEST_FLAGS) $(BUILD_JOBS)
+ifneq ($(TEST_CMD),test)
+ifneq ($(HAS_DOCTEST_TARGETS),)
+	RUSTFLAGS="$(RUST_FLAGS)" $(CARGO) test --doc --workspace --all-features
+endif
+endif
 
 target/%/$(TARGET): ## Build binary in debug or release mode
 	$(CARGO) build $(BUILD_JOBS) $(if $(findstring release,$(@)),--release) --bin $(TARGET)
 
 lint: ## Run Clippy with warnings denied
```

checkmake (0.2.2)

[warning] Target body for "lint" exceeds allowed length of 5 (7).

(maxbodylength)